src/main/java/com/github/princesslana/slothbot/MessageCounter.java

```diff
@@ -81,6 +106,53 @@ public void clearBuckets(String channelId) {
                 .collect(ImmutableList.toImmutableList()));
   }
 
+  public boolean getBotCounterConfig(String guildId) {
+    return Optional.ofNullable(botCounterConfigs.get(guildId)).orElse(false);
+  }
+
+  public void setBotCounterConfig(String guildId, boolean value) {
+    if (value) {
+      botCounterConfigs.put(guildId, true);
+    } else {
+      botCounterConfigs.remove(guildId);
+    }
+    save();
+  }
+
+  private void save() {
+    var json = Json.array();
+
+    botCounterConfigs.forEach(
+        (k, v) -> json.add(Json.object().add("guild_id", k).add("count_bot_msgs", v)));
+
+    try {
+      MoreFiles.asCharSink(savePath, Charsets.UTF_8)
+          .write(json.toString(WriterConfig.PRETTY_PRINT));
+      LOG.atDebug().log("Saved {} counter configs to {}", json.size(), savePath);
+    } catch (IOException e) {
+      LOG.atWarn().withThrowable(e).log("Error saving counter configs to {}", savePath);
+    }
+  }
+
+  public void load() {
```

Comment:
Where is load called?

Reply:
Oh right, I forgot to call it, I'll change that.